desktop/apps/artwork/components/partner_stub/components/inquire_via_phone_modal.coffee

```diff
@@ -0,0 +1,29 @@
+AuthModalView = require '../../../../../components/auth_modal/view.coffee'
+{ templateMap } = require '../../../../../components/auth_modal/maps.coffee'
+mediator = require '../../../../../lib/mediator.coffee'
+
+templateMap['phone_number'] = -> require('../templates/phone_number.jade') arguments...
+
+module.exports = class InquireViaPhoneModalView extends AuthModalView
+  template: ->
+    templateToRender = if sd.CURRENT_USER then 'phone_number' else @state.get('mode')
+    templateMap[templateToRender] arguments...
+
+  onSubmitSuccess: (model, response, options) =>
+    analyticsHooks.trigger "auth:#{@state.get 'mode'}"
```

Comment:
Actually, we might be able to add a context without adding a call in `onSubmitSccess`. Really the call I wanted to avoid caliling is just [`@$('form').submit()`](https://github.com/artsy/force/blob/fa899e0/desktop/components/auth_modal/view.coffee#L133). If we could avoid it and just call `super`, I'd be more happy to do so.